dto/src/main/java/org/jboss/pnc/rex/dto/ConfigurationDTO.java

```diff
@@ -33,5 +33,6 @@
 @ToString
 public class ConfigurationDTO {
 
-    public boolean passResultsOfDependencies = false;
+    private boolean passResultsOfDependencies = false;
+    private boolean skipCallback = false;
```

Comment:
can you split the configuration for start and cancel? I can see someone would want fast cancel but regular start with callback